src/Disks/ObjectStorages/MetadataStorageFromPlainObjectStorageOperations.cpp

```diff
@@ -418,11 +418,11 @@ MetadataStorageFromPlainObjectStorageMoveFileOperation::MetadataStorageFromPlain
     , object_storage(object_storage_)
 {
     {
-        auto tmp_path_from = path_to.string() + ".move_from." + getRandomASCIIString(16);
+        auto tmp_path_from = path_to.string() + "." + getRandomASCIIString(16) + ".tmp_move_from";
```

Comment:
Do we care about leftover files with names in the old format, which will not be removed because they don't end with ".tmp_move_from" (or ".tmp_move_to")?